Configuration/EventContent/python/EventContent_cff.py

```diff
@@ -640,6 +641,19 @@ def SwapKeepAndDrop(l):
 FEVTDEBUGHLTEventContent.outputCommands.append('keep *_*_MergedTrackTruth_*')
 FEVTDEBUGHLTEventContent.outputCommands.append('keep *_*_StripDigiSimLink_*')
 FEVTDEBUGHLTEventContent.outputCommands.append('keep *_*_PixelDigiSimLink_*')
+
+pp_on_PbPb_run3.toModify(FEVTDEBUGHLTEventContent,
```

Comment:
I'm wondering if it would make sense to customize
https://github.com/cms-sw/cmssw/blob/994bf20b343025de0263cde37f7b3fcbdde5c17e/HLTrigger/Configuration/python/HLTrigger_EventContent_cff.py#L219
instead?

Reply:
No,  because that implies a charge in the confdb menu and I don't think we want this at this point (perhaps later)

Reply:
> I'm wondering if it would make sense to customize
[cmssw/HLTrigger/Configuration/python/HLTrigger_EventContent_cff.py](https://github.com/cms-sw/cmssw/blob/994bf20b343025de0263cde37f7b3fcbdde5c17e/HLTrigger/Configuration/python/HLTrigger_EventContent_cff.py#L219)

For the record (while a bit inelegant) this was taken care in the development combined table at https://its.cern.ch/jira/browse/CMSHLT-2964. This should not stop integration of these changes though, as the ad-hoc customization can be removed once the HLT menu is updated in the release.

Reply:
this has been taken care of by https://github.com/cms-sw/cmssw/pull/43300 (and corresponding backports)